pkg/upgrade/plan/plan.go

```diff
@@ -168,11 +168,13 @@ func SelectConcurrentNodeNames(plan *upgradeapiv1.Plan, nodeCache corectlv1.Node
 			jsum := sha256sum(string(candidateNodes[j].UID), string(plan.UID), plan.Status.LatestHash)
 			return isum < jsum
 		})
+		for _, node := range applying {
+			selected = AppendIfMissing(selected, node)
+		}
```

Comment:
I do not see why this block is necessary given the following (from above):
```
		applyingNodes, err := nodeCache.List(nodeSelector.Add(*requirementApplying))
		if err != nil {
			return nil, err
		}
		for _, node := range applyingNodes {
			selected = append(selected, node.Name)
		}
```

Additionally, I believe that this will prevent an "applying" node from ever dropping off of the "selected" slice returned by this func.